Automattic/wp-calypso
# WooExpress: Update trial "Plans"page accordion

## Proposed Changes

* This PR changes the way the accordion works in the Plans page (for the eCommerce trial)

Now the whole header triggers the toggle, not just the arrow on the right:

Before:
![image](https://user-images.githubusercontent.com/3801502/217897684-01508852-f3a2-41f2-800b-6efbf1cc157d.png)

After:
![image](https://user-images.githubusercontent.com/3801502/217897825-1d444768-0571-4e50-8b26-1c4d9f407404.png)


## Testing Instructions

* Open the Plans page of a site on the eCommerce trial plan (`/plans/<site-slug>`)
* Check that you can toggle the accordion by clicking on any part of the header in both Desktop and Mobile versions.
